src/test/java/com/cloudant/tests/CloudantClientTests.java

```diff
@@ -66,7 +67,6 @@ public void setUp() {
     @After
     public void tearDown() {
         account.shutdown();
-        cookieBasedClient.shutdown();

```

Comment:
I don't think this should be removed, since looking at the setup code the cookieBasedClient isn't the same as account.